applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -496,7 +498,7 @@ class ExplicitCentralDifferencesScheme
         const IndexType initial_k = DomainSize == 3 ? 0 : 2; // We do this because in 2D only the rotation Z is needed, then we start with 2, instead of 0
         for (IndexType kk = initial_k; kk < 3; ++kk) {
             if (nodal_inertia[kk] > numerical_limit)
-                r_current_angular_acceleration[kk] = r_current_residual_moment[kk] / nodal_inertia[kk];
+                r_current_angular_acceleration[kk] = (r_current_residual_moment[kk] - nodal_rotational_damping[kk] * r_current_angular_velocity[kk]) / nodal_inertia[kk];
```

Comment:
this is not as simple as this... you cannot lump the Inertia tensor, unless you work in the principal system of coordinates

Reply:
That hypothesis was already there, ask @KlausBSautter for details

Reply:
@KlausBSautter this is only ok if you update the inertia when you rotate the beam local axes, and addionally in the hypothesis of working in the principle system of intertia of the beam. otherwise you are doing a major error with this